### ISSUE TITLE
file_get_contents Issue

### ISSUE THREAD

**Issue by satrya** [satrya/of-backup#1]
Hi @devinsays can you help me with creating options backup for Options Framework. I actully adapt the code from
1. https://github.com/vauvarin/options-framework-theme/blob/master/admin/options-backup.php
2. https://pippinsplugins.com/building-settings-import-export-feature/

The problem is, this code using `file_get_contents` [#L163](https://github.com/satrya/of-backup/blob/master/class-options-backup.php#L163) functions which is not allowed, do you know which WordPress core function to replace `file_get_contents`?

**Comment by devinsays** [comment]
Have you seen @helgatheviking's version?
https://github.com/helgatheviking/options-framework-importer

**Comment by satrya** [comment]
I actually would like to include the backup options in the theme. I saw `base64_encode` code in @helgatheviking's version. Do you have another opinions?

**Comment by devinsays** [comment]
base64_encode makes sense for something like this. Is this a theme you're trying to get in the repository?

**Comment by satrya** [comment]
No, it's a commercial theme. Does `base64_encode` passed the theme check?

**Comment by devinsays** [comment]
It doesn't. Generally that function is used to obfuscate code, but in this instance it's used for valid reasons.

I'm sure there is a workaround though if you prefer not to use that function.

**Comment by helgatheviking** [comment]
Wow, i totally forgot that i had posted that on github. Anyway, once upon a time I think i tried to convert it to json? I don't remember and then I told myself to stop working on it because I don't really do much theme things any more. You could certainly fork what I have.

**Comment by satrya** [comment]
Thank you @devinsays 

@helgatheviking you version is better, I'll consider using yours code. Thank you
